### ISSUE TITLE
Trying to understand the loss function

### ISSUE THREAD

**Issue by LucasColas** [ark1234/NeurIPS2023-InfoCD#2]
Hello,
It seems the code of InfoCD is here : https://github.com/Zhang-VISLab/NeurIPS2023-InfoCD/blob/main/codes/utils/loss_utils.py#L91 
However, it's really different from the formula proposed here : https://proceedings.neurips.cc/paper_files/paper/2023/file/f2ea1943896474b7cd9796b93e526f6f-Paper-Conference.pdf isn't it ?
The code attempts exp(-0.5 * d) / sum(...) but then mangles the denominator; it should divide by the plain sum.
Why is it multiplied by 1e3 ?
Why return (torch.sum(distances1) + torch.sum(distances2)) / 2 in calc_cd_like_InfoV2 ?
